lib/compat/wordpress-5.9/class-gutenberg-rest-templates-controller.php

```diff
@@ -236,7 +243,7 @@ public function update_item_permissions_check( $request ) {
 	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
 	 */
 	public function update_item( $request ) {
-		$template = gutenberg_get_block_template( $request['id'], $this->post_type );
+		$template = gutenberg_get_block_template_with_fallback( $request['id'], $this->post_type );
```

Comment:
I couldn't figure out how to put something like

```suggestion
		if ( isset( $request['resolve'] ) && true === $request['resolve'] ) {
			$template = gutenberg_get_block_template_with_fallback( $request['id'], $this->post_type );
		} else {
			$template = gutenberg_get_block_template( $request['id'], $this->post_type );
		}
```

(i.e. akin to [what we do](https://github.com/WordPress/gutenberg/pull/37258/files#diff-c16d9630166c1e0ad1ded82c9a9c1e69564de3ccd533726d1c843309ead21c34R216-R217) in `get_item`), since `update_item` is a POST request (`WP_REST_Server::EDITABLE`), and its `args` are defined by `$this->get_endpoint_args_for_item_schema( WP_REST_Server::EDITABLE )` -- which basically reflects the attributes of a template object but doesn't leave room for a `resolve` bool 🤔